test-case-reporting/src/travis_api_utils.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
If this isn't used in this PR, let's stash it away on another branch until it's needed.

Reply:
Done.